test/controllers/order_items_controller_test.rb

```diff
@@ -0,0 +1,81 @@
+require "test_helper"
+
+describe OrderItemsController do
+  describe "index" do
+    it "sends a success response when there are many order items" do
+      get order_items_path
+
+      must_respond_with :success
+    end
+
+    it "sends a success response when there are no orders" do
+      OrderItem.destroy_all
+      get order_items_path
+      must_respond_with :success
+    end
+  end
+  describe "show" do
+    it "should get show" do
+      get order_item_path(OrderItem.first.id)
+
+      must_respond_with :success
+    end
+
+  end
+
+  describe "create" do
+
+    it "sends success if the order exists" do
+      order = Order.first
+      orderitem_data = { product_id: Product.first.id, quantity: Product.first.stock, order_id: order.id }
+
```

Comment:
You should probably have tests for both adding a new product to the cart, and adding a product that's already in the cart (update the quantity)